src/Microsoft.DotNet.Metrics/Microsoft.DotNet.Metrics.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <RuntimeIdentifier>win7-x64</RuntimeIdentifier>
```

Comment:
Why does this have a RID in it? This package should have no problem working on all platforms.

Reply:
I just copied the "Shared" project. I have no idea what like 90% of these things do. :-)